DecipherNow/helm-charts
# Fabric chart cleanup

Changes
- Use a different proxy kubernetes label name (requested by @chrisbsmith). Instead of app, we're using “gm.proxy” 
- Update proxy port to use `10808` (another @chrisbsmith request)
- Use snake case
- Rename gm-control-api to control-api
- Add comments to all values files
- Expose all envvars in values files, remove from template files
- Rip out spire
- Clean up all global values.

To test:
1. Install secrets
```
helm install secrets secrets --set=global.environment=kubernetes
```
2. Install each chart by itself. Make sure there are no errors on install. (Control will not come up by itself, but control-api should have 2/2 pods ready)
```
helm install fabric fabric/control --set=global.environment=kubernetes
helm uninstall fabric
helm install fabric fabric/control-api --set=global.environment=kubernetes
helm uninstall fabric
```
3. Install the fabric umbrella chart. Make sure there are no errors on install and that all pods come up.
```
helm dep up fabric
helm install fabric fabric --set=global.environment=kubernetes
```

```
$ kubectl get pods
NAME                       READY   STATUS    RESTARTS   AGE
control-669667cf9d-rx7fs   1/1     Running   0          70s
control-api-0              2/2     Running   0          29s
```
